packages/blockchain/src/blockchain.ts

```diff
@@ -390,7 +380,13 @@ export class Blockchain implements BlockchainInterface {
         }
 
         if (this._validateConsensus) {
-          await this.consensus.validateConsensus(block)
+          if (!(this.common.consensusAlgorithm() in this._consensusDict)) {
```

Comment:
For cleani-ness should this not use `this.consensus()`?

Reply:
You are right, that works as well 🙂, will update!